src/app/download/download.service.ts

```diff
@@ -10,6 +11,15 @@ export class DownloadService {
     private readonly adAuthoringState: AdAuthoringWorkflowStateContainer
   ) {}
 
+  downloadDisabled() {
```

Comment:
Since the function is returning a boolean, can we change the name to isDownloadDisabled for clarity?